class.jetpack-jitm.php

```diff
@@ -321,7 +321,7 @@ function backups_updates_msg() {
 			<a href="#" data-module="vaultpress" class="dismiss"><span class="genericon genericon-close"></span></a>
 
 			<div class="jp-emblem">
-				<?php echo self::get_jp_emblem(); ?>
+				<?php echo Jetpack::get_jp_emblem(); ?>
```

Comment:
Instead of adding one more thing to the Jetpack class. Why not have this code be part of the `Jetpack_JITM one? 

Also it seems that we are always repeating `<div class="jp-emblem">` before each function call. Could that be part of the method instead? 

Also I think renaming the method to just `get_emblem` would be a bit more meaningful.